panel-ui.js

```diff
@@ -159,22 +177,22 @@ class App extends Component {
               className: 'first' + (this.state.active.prevState ? ' selected' : ''),
               onClick: () => this.toggleActive('prevState')
             }, [
-              '{', e(FontAwesome, { name: 'arrow-circle-o-left', title: 'View Previous State' }), '}'
```

Comment:
Looks like you've got some extra whitespace here. That might be part of what's causing the conflict.